### PR TITLE
test vibe 0.8.1-beta.1

### PR DESCRIPTION
**Pull request by MartinNowak**
- to help https://github.com/dlang/dmd/pull/6998

### CODE REVIEW

File: pipeline.groovy [hunk 1]
```diff
@@ -147,7 +147,12 @@ def testDownstreamProject (name) {
                     "HOME=${env.WORKSPACE}"
                 ]) {
             try { dir(repo) {
-                cloneLatestTag("https://github.com/${repo}.git")
+
+                if (repo == 'rejectedsoftware/vibe.d') {
+                    clone("https://github.com/${repo}.git", 'v0.8.1-beta.1')
```
Comment (wilzbach) [inline pipeline.groovy]: Seems like the vibeVersionString wasn't properly updated :/

**Comment by MartinNowak** [comment]
Could you please tag vibe.d v0.8.1-beta.2 and use [`./scripts/tag_version.sh`](https://github.com/rejectedsoftware/vibe.d/blob/82d5503b60b0c685658542636c979b6161209254/scripts/tag_version.sh) to do so @s-ludwig.

**Comment by s-ludwig** [comment]
Damn.. I always forget that...

**Comment by ZombineDev** [comment]
@MartinNowak `v0.8.1-beta.2` was tagged, so we can proceed here.

**Pull request merged**